packages/nimble-components/src/rich-text-mention/users/index.ts

```diff
@@ -0,0 +1,73 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { MentionInternalsOptions } from '../base/models/mention-internals';
+import {
+    MappingConfigs,
+    RichTextMention,
+    RichTextMentionConfig
+} from '../base';
+import type { MappingConfig } from '../base/models/mapping-config';
+import { MappingUserConfig } from '../base/models/mapping-user-config';
+import { template } from '../base/template';
+import { iconAtTag } from '../../icons/at';
+import { MappingMentionUser } from '../../mapping/mention-user';
+import type { Mapping } from '../../mapping/base';
+import type { MentionHref } from '../../mapping/base/types';
+import { RichTextMentionUsersValidator } from './models/rich-text-mention-users-validator';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-rich-text-mention-users': RichtextMentionUsers;
+    }
+}
+
+/**
+ * Rich Text Mention that will map user url and name
+ */
+export class RichtextMentionUsers extends RichTextMention<
```

Comment:
Use consistent capitalization: `RichText`, not `Richtext`. Same for the variable further down this file.

Reply:
Updated